src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -101,7 +101,7 @@ const Body = BaseFormWithPolling.extend(Object.assign(
 
     remove() {
       BaseFormWithPolling.prototype.remove.apply(this, arguments);
-      this.stopPolling();
+      this.stopPoll();
```

Comment:
it just not working with `stopPolling`?

Reply:
Yes. `stopPoll` calls `stopPolling` + `clearTimeout`